app/src/main/java/org/piwigo/ui/login/LoginActivity.java

```diff
@@ -76,13 +77,18 @@
     private AccountAuthenticatorResponse authenticatorResponse;
     private Bundle resultBundle;
 
+    public CountingIdlingResource idleResource;
```

Comment:
ah, we have many IdlingResources... Maybe better document their use in the README?

Reply:
and why is this idleResource here not inc-/decremented? - Not that I want to say it would be necessary, I just want to understand it...